src/sentry/api/endpoints/auth_login.py

```diff
@@ -26,7 +26,9 @@ def post(self, request, organization=None, *args, **kwargs):
         # Rate limit logins
         is_limited = ratelimiter.is_limited(
             u"auth:login:username:{}".format(
-                md5_text(request.data.get("username").lower()).hexdigest()
+                md5_text(
+                    login_form.clean_username(request.data.get("username").lower())
```

Comment:
I guess you don't need `lower()` here now since it's going through clean. And in the other place.